modules/extension/wmts/src/test/java/org/geotools/ows/wmts/WebMapTileServerTest.java

```diff
@@ -72,9 +72,7 @@ public void chGetEnvelopeTest() throws Exception {
         // <ows:UpperCorner>11.47757 48.230651</ows:UpperCorner>
         // </ows:WGS84BoundingBox>
 
-        assertNotNull(wmts.getEnvelope(layer, CRS.decode("urn:ogc:def:crs:OGC:1.3:CRS84")));
-        assertNotNull(wmts.getEnvelope(layer, CRS.decode("CRS:84")));
-        assertNotNull(wmts.getEnvelope(layer, CRS.decode("EPSG:2056")));
+        assertNotNull(wmts.getEnvelope(layer, CRS.decode("urn:ogc:def:crs:EPSG:2056")));
```

Comment:
What's the rational for removing these additional checks?

Reply:
Sorry, accidental, will put them back in